### DISCUSSION TITLE
Using Azure signalr core from mono (raspberry pi)

### DISCUSSION THREAD

**Discussion by michaelonz** [dotnet/aspnetcore#27325] [Q&A]
Hi Guys,
    I have a server webapplication using signalr on azure.   I can call methods on the serverhub (and back again) using the Microsoft.AspNetCore.SignalR.Client package running as a client on windows 10.  This seems to work fine.  However i now need to receive notifications from the azure server app on a raspberry pi running mono.  I have tried to use the Microsoft.AspNetCore.SignalR.Client (this doesnt work because it needs core and the app i have is a windows forms app so i cant use core).  I have also tried using Microsoft.AspNet.SignalR.Client package  - this errors and i have tried many combinations...including other code such as https://github.com/mdymel/mono-signalr-client/tree/master/signalr 

Should i be able to do this - does anyone have a sample of an azure web app using core on the server and the .net 4.5 client framework calling/receiving notifications?

Regards

Mike

**Comment by BrennanConroy** [comment]
ASP.NET SignalR is not compatible with ASP.NET Core SignalR.

The ASP.NET Core SignalR client targets netstandard2.0 which should be able to run on .NET Framework.
